### ISSUE TITLE
Extract definitions from names.py to config files

### ISSUE THREAD

**Issue by orelgueta** [gammasim/simtools#224]
Extract the definitions from names.py for the additional flexibility it would provide and more generic code it would produce.

**Comment by GernotMaier** [comment]
Agree that we want to move the naming dicts out of the code probably into yaml files. 

We don't have a space for these files, would that be a new subdirectory? Maybe `data/names.yml`

And just to be sure, we are talking about those fields?

```
lst = "LST"
mst = "MST"
sct = "SCT"
sst = "SST"
hess = "HESS"
magic = "MAGIC"
veritas = "VERITAS"

all_telescope_class_names = {
    lst: ["lst"],
    mst: ["mst"],
    sct: ["sct"],
    sst: ["sst"],
    hess: ["hess"],
    magic: ["magic"],
    veritas: ["veritas"],
}

all_camera_names = {
    "SST": ["sst"],
    "ASTRI": ["astri"],
    "GCT": ["gct", "gct-s"],
    "1M": ["1m"],
    "FlashCam": ["flashcam", "flash-cam"],
    "NectarCam": ["nectarcam", "nectar-cam"],
    "SCT": ["sct"],
    "LST": ["lst"],
}
```

Also note here that DPPS / ACADA also introduce a numerical telescope identifier, see [this gitlab page](https://gitlab.cta-observatory.org/cta-computing/common/identifiers/-/blob/main/array-element-ids.json?ref_type=heads)

**Comment by GernotMaier** [comment]
I am wondering if we still want to do that: having the name definition in parameter files is not much different to having them directly in the code. I would rather move towards using the database for certain entries in name.py (especially for all_model_version_names).

**Comment by VictorBarbosaMartins** [comment]
For the dictionaries I agree it might be useful to use the DB, but for such simple parameter name definition such as `lst = "LST"` we don't need to overcomplicate and get it from the database.

**Comment by GernotMaier** [comment]
We have discussed this again in the context of issue #797 and PR #761 , and introduced a simpler naming of telescopes. I suggest therefor to close this issue and leave the naming as dictionaries in names.py.

**Comment by GernotMaier** [comment]
This has been addressed in #889 - all telescope naming is removed from names.py
